Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1296,15 +1397,15 @@ Lastly, see if you can play with the version number of a man-in-the-middled app
 
 #### OWASP MASVS
 
-- V1.9: "A mechanism for enforcing updates of the mobile app exists."
-- V6.1: "The app only requests the minimum set of permissions necessary."
-- V6.2: "All inputs from external sources and the user are validated and if necessary sanitized. This includes data received via the UI, IPC mechanisms such as intents, custom URLs, and network sources."
-- V6.3: "The app does not export sensitive functionality via custom URL schemes, unless these mechanisms are properly protected."
-- V6.4: "The app does not export sensitive functionality through IPC facilities, unless these mechanisms are properly protected."
-- V6.5: "JavaScript is disabled in WebViews unless explicitly required."
-- V6.6: "WebViews are configured to allow only the minimum set of protocol handlers required (ideally, only https is supported). Potentially dangerous handlers, such as file, tel and app-id, are disabled."
-- V6.7: "If native methods of the app are exposed to a WebView, verify that the WebView only renders JavaScript contained within the app package."
-- V6.8: "Object serialization, if any, is implemented using safe serialization APIs."
+- MSTG-ARCH-9: "A mechanism for enforcing updates of the mobile app exists."
```

Comment:
maybe put it at the end of this list as it also appears at the end of the chapter